resources/text/units.md

```diff
@@ -0,0 +1,96 @@
+# Units
+
+Written because the author confused themselves and wanted to be sure they
+now understand.
+
+There are three main classes of coordinates in fonts:
```

Comment:
``diff
- There are three main classes of coordinates in fonts:
+ There are three main classes of axis coordinates in variable fonts:
```

Reply:
Will apply that edit directly as the merge button has gone away

Reply:
Doen in https://github.com/googlefonts/fontmake-rs/commit/67b44d2b88e0daaac9680c5f40df0d062d0676f0